solana/gatekeeper-lib/src/util/connection.ts

```diff
@@ -102,41 +102,11 @@ export class SendableTransaction implements TransactionHolder {
     return new SentTransaction(this.connection, txSig);
   }
 
-  /**
-   * Message is separate from signatures due to the way rebuilding serialized transactions doesn't handle signatures
-   */
-  serializeForRelaying(): {
-    message: Buffer;
-    signatures: string[];
-  } {
-    const message = this.transaction.compileMessage();
-    const signatures = message.accountKeys
-      .slice(0, message.header.numRequiredSignatures)
-      .map((key) => {
-        const result = this.transaction.signatures.find((sig) =>
-          sig.publicKey.equals(key)
-        );
-        if (result && result.signature) {
-          return bs58.encode(result.signature);
-        } else {
-          return bs58.encode(DEFAULT_SIGNATURE);
-        }
-      });
-    return {
-      message: message.serialize(),
-      signatures,
-    };
-  }
-
   static fromSerialized(
     connection: Connection,
-    message: Buffer,
-    signatures: string[]
```

Comment:
Missing context, but why was `signatures` removed here?

Reply:
The signatures are now in the buffer, it's the solana way of doing it where the whole transaction is serialized